fails to run on fresh install

```
> octagon.react@0.1.0 start /Users/cdieringer/node/octagon.react
> patternplate start --open

⠹ Starting/Users/cdieringer/node/octagon.react/node_modules/patternplate/binary/patternplate.js:182
                throw error;
                ^

Error: Cannot find module 'patternplate-transform-browserify'
...
```

--- a/readme.md
+++ b/readme.md
@@ -26,13 +26,13 @@ Get up and running with patternplate in three steps. You can find the source of 
 
 ```shell
 # Grab patternplate from npm
-npm install --save patternplate
+npm install -g patternplate
 
 # Create an initial patternplate setup
-./node_modules/.bin/patternplate init
+patternplate init patternplate-project
 
 # Start patternplate and open in default browser
-npm start -- --open
+cd patternplate-project && npm install && npm start -- --open
 
 # Go forth and create, show, deliver
 ```

--- a/source/library/init/index.js
+++ b/source/library/init/index.js
@@ -41,7 +41,6 @@ async function init(directory = '.', options) {
 	const resolve = (...args) => path.resolve(...[cwd, ...args]);
 
 	const manifestPath = resolve(settings.manifestPath);
-	const nodeModulesPath = resolve(cwd, 'node_modules');
 	const name = path.basename(cwd);
 
 	// Add a name based on directory if manifest does not exists
@@ -85,7 +84,7 @@ async function init(directory = '.', options) {
 
 	const instructions = [
 		process.cwd() !== cwd && `cd ${directory}`,
-		!await exists(nodeModulesPath) && 'npm install',
+		'npm install',
 		data.scripts.start === 'patternplate' ?
 			'npm start -- --open' : './node_modules/.bin/patternplate start --open'
 	].filter(Boolean);

